README.md

```diff
@@ -122,13 +122,18 @@ mergeStream(project.sources(), project.dependencies())
 The bundler() method accepts an options object to configure bundling.  See [Using polymer-bundler programmatically](https://github.com/polymer/polymer-bundler#using-polymer-bundler-programmatically) for a detailed list of accepted options.
 
 ```js
+const {generateCountingSharedBundleUrlMapper,
+       generateSharedDepsMergeStrategy} = require('polymer-bundler');
+
+mergeStream(project.sources(), project.dependencies())
   .pipe(project.bundler({
     excludes: ['bower_components/polymer-code-mirror'],
     sourcemaps: true,
     stripComments: true,
-    strategy: require('polymer-bundler/lib/bundle-manifest')
-      .generateSharedDepsMergeStrategy(3)
+    strategy: generateSharedDepsMergeStrategy(3),
+    urlMapper: generateCountingSharedBundleUrlMapper('shared/bundle_')
```

Comment:
is there a default behavior in polymer-bundler if none is provided? This is getting a bit into how the sausage is made, which is fine if its required but adds confusion unnecessarily if this isn't necessary for most people.